content/90.back-matter.md

```diff
@@ -2,6 +2,25 @@
 
 The authors thank [Blair Sullivan](https://orcid.org/0000-0001-7720-6208) for [her feedback](https://github.com/greenelab/xswap-manuscript/issues/54) on a draft of the manuscript.
 
+## Availability of Supporting Source Code and Requirements
```

Comment:
I found the best way to ensure we don't get subsequent copyedit changes is to look at other recent [TECHNICAL NOTES](https://academic.oup.com/gigascience/search-results?sort=Date+%e2%80%93+Newest+First&f_ContentType=Journal+Article&f_TocHeadingTitle=TECHNICAL+NOTE&fl_SiteID=5589&page=1):

- https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giad104/7478178?searchresult=1#431395973
- https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giad101/7471725?searchresult=1#430106498
- https://academic.oup.com/gigascience/article/doi/10.1093/gigascience/giad108/7471724?searchresult=1#430106336

I think this section looks mostly good based on these examples

Reply:
Currently, we each item here is its own paragraph. We probably want to make each its own line in a single paragraph. From [pandoc docs](https://pandoc.org/MANUAL.html#extension-escaped_line_breaks):

>  If you need a hard line break, put two or more spaces at the end of a line. A backslash followed by a newline is also a hard line break. 

A third method is to put `<br>` at the end of each line.